lib/apiservers/engine/backends/container.go

```diff
@@ -482,8 +493,9 @@ func (c *ContainerBackend) taskStateWaitHelper(op trace.Operation, id, eid, name
 // ContainerExecStart starts a previously set up exec instance. The
 // std streams are set up.
 func (c *ContainerBackend) ContainerExecStart(ctx context.Context, eid string, stdin io.ReadCloser, stdout, stderr io.Writer) error {
-	op := trace.FromContext(ctx, "exec start")
-	defer trace.End(trace.Begin("", op))
+	op := trace.NewOperation(context.Background(), "ContainerExecStart: %s", eid)
```

Comment:
Use `trace.FromContext` - I doubt this is a `trace.Operation` but seeing as we have a context passed in we should wrap it.